js/app/components/mobile/desktop-ui/top-controls.tsx

```diff
@@ -97,9 +106,20 @@ export function TopControlBar({
           </>
         )}
         {ingest !== null && (
-          <Pressable onPress={doSetIngestCamera} style={[p[2], r[1]]}>
-            <SwitchCamera size={24} color={colors.gray[200]} />
-          </Pressable>
+          <>
+            {ingestMediaSource !== IngestMediaSource.DISPLAY && (
+              <Pressable onPress={doSetIngestCamera} style={[p[2], r[1]]}>
+                <SwitchCamera size={24} color={colors.gray[200]} />
+              </Pressable>
+            )}
+            <Pressable onPress={toggleMediaSource} style={[p[2], r[1]]}>
+              {ingestMediaSource === IngestMediaSource.DISPLAY ? (
+                <Webcam size={24} color={colors.gray[200]} />
+              ) : (
+                <Monitor size={24} color={colors.gray[200]} />
+              )}
+            </Pressable>
+          </>
```

Comment:
should probably reverse these two so webcam/monitor is always on top
```suggestion
          <>
            <Pressable onPress={toggleMediaSource} style={[p[2], r[1]]}>
              {ingestMediaSource === IngestMediaSource.DISPLAY ? (
                <Webcam size={24} color={colors.gray[200]} />
              ) : (
                <Monitor size={24} color={colors.gray[200]} />
              )}
            </Pressable>
            {ingestMediaSource !== IngestMediaSource.DISPLAY && (
              <Pressable onPress={doSetIngestCamera} style={[p[2], r[1]]}>
                <SwitchCamera size={24} color={colors.gray[200]} />
              </Pressable>
            )}
          </>
```